README.md

```diff
@@ -98,6 +100,7 @@ ignore_missing=true
 | upload_debug_symbols | Enables or disables the automatic upload of debug symbols | true (boolean) | no | - |
 | upload_source_maps | Enables or disables the automatic upload of source maps | false (boolean) | no | - |
 | upload_sources | Does or doesn't include the source code of native code | false (boolean) | no | - |
+| legacy_web_symbolication | Uses legacy symbolication method for Flutter Web instead of Debug IDs (requires Sentry Flutter SDK < 9.1.0) | false (boolean) | no | - |
```

Comment:
just a small correction, you can still use the legacy one with newer sentry flutter sdk versions as well, its not limited to older versions